meta-psa/README.md

```diff
@@ -0,0 +1,21 @@
+# meta-psa OpenEmbedded layer
+
+The meta-psa OpenEmbedded layer provides reference implementations and tests of
+the Arm Platform Security Architecture APIs.
+
+## PSA
+The Arm Platform Security Architecture (PSA) is a holistic set of threat
+models, security analyses, hardware and firmware architecture specifications,
+and an open source firmware reference implementation. PSA provides a recipe,
+based on industry best practice, that allows security to be consistently
+designed in, at both a hardware and firmware level.
+
+For more information, see the [PSA website][psa].
+
+## Recipes
```

Comment:
If there's just the one, please don't use a bulleted list.

Reply:
There will be more very soon. https://github.com/ARMmbed/meta-mbl/pull/598 adds a new recipe (psa-arch-tests) in this layer.

I suppose maybe I should just list psa-arch-tests in the list and wait until https://github.com/ARMmbed/meta-mbl/pull/598 is merged before merging this one.

Reply:
Added reference to psa-arch-tests recipe in https://github.com/ARMmbed/meta-mbl/pull/600/commits/bba62cdb08b673c41084bb26ea2fcfec50e4f8d7.